addons/web/controllers/webclient.py

```diff
@@ -64,6 +64,7 @@ def qweb(self, unique, mods=None, db=None, bundle=None):
                 ('Cache-Control', f'public, max-age={http.STATIC_CACHE_LONG}')
             ])
 
+    # FP TODO 4: Cache the method, load all .mo (gettext), or .po (babel) if no .mo
     @http.route('/web/webclient/bootstrap_translations', type='json', auth="none")
```

Comment:
Looks like two implementations of the same thing;
- one in ir.translations.code
- this one

I would put your code in ir.Translation.code here, to have only one approach to fetch web client translations.